samples/SocialSample/SocialSample.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
```

Comment:
Have we looked at consolidating the sample TFMs in a Directory.Build.props?

Reply:
That would be awesome if you want to make an issue/PR